### ISSUE TITLE
lint: multi-value case causes emptyFallthrough false positives

### ISSUE THREAD

**Issue by quasilyte** [go-critic/go-critic#645]
This pattern of splitting long single-line case clauses is quite common:
```go
	case token.PLUS, token.MINUS, token.NOT, token.BITWISE_NOT:
		fallthrough
	case token.DELETE, token.VOID, token.TYPEOF:
```
`emptyFallthrough` should skip these by default.

**Comment by Quasilyte** [comment]
Probably this is another special case:
```go
	case 'B':
		fallthrough

	case 'd', 'D', 's', 'S', 'w', 'W':
		// This is slightly broken, because ECMAScript
		// includes \v in \s, \S, while re2 does not
		fallthrough

	case '\\':
		fallthrough

	case 'f', 'n', 'r', 't', 'v':
```
When several distinct groups are matched together. 
It can be re-written with a single case, but this would be opinionated thing that depend on the project-local conventions.
